forum/qa-plugin/report-reason/q2apro-flag-reasons-admin.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+class q2apro_flagreasons_admin
+{
+    const NOTICE_LENGTH = 255;
```

Comment:
to w sumie można zrobić konfigurowalne przez ustawienia wtyczki i zapisywalne w bazie. Taki #protip

Reply:
Nie widzę sensu trzymać tego w bazie jako ustawienie, bo to raczej nie będzie zmieniane, nie za bardzo jest sens tym manipulować po prostu. Do tego jeszcze długość powodu jest ograniczona wielkością kolumny w bazie, więc albo tu musiałoby być ograniczenie wielkości w konfiguracji wtyczki do wielkości kolumny albo jeszcze konfiguracja wtyczki musiałaby zmieniać kolumnę w bazie. Prościej wg mnie zostawić to jak jest